### ISSUE TITLE
cho_solve is not available on the `linalg` namespace.

### ISSUE THREAD

**Issue by zoj613** [aesara-devs/aesara#708]
We cannot import `cho_solve` via the `linalg` namespace. One must use `slinalg`. This is because it is not included in the `__all__` list of the module.

**Please provide the full traceback of any errors.**
Python says `cho_solve` is not available if accessed via `at.linalg.cho_solve`.

## Versions and main components

* Aesara version: latest
* Python version: 3.9
* Operating system: Arch
* How did you install Aesara: pip

**Comment by brandonwillard** [comment]
We're using `aesara.tensor.linalg` and `aesara.tensor.slinalg` to distinguish between `numpy.linalg` and `scipy.linalg`, respectively.  We can consider merging both interfaces, though; however, we would need to consider what to do about name clashes (if there are any).

**Comment by zoj613** [comment]
> We're using `aesara.tensor.linalg` and `aesara.tensor.slinalg` to distinguish between `numpy.linalg` and `scipy.linalg`, respectively. We can consider merging both interfaces, though; however, we would need to consider what to do about name clashes (if there are any).

It seems like both `nlinalg` and `slinalg` are being imported into the same `linalg` namespace:
https://github.com/aesara-devs/aesara/blob/ff4b91dddede43d63d348bdec8fe1079932146b0/aesara/tensor/linalg.py#L1-L2

So I assumed functions from both modules are meant to accessed in the unified namespace.

**Comment by brandonwillard** [comment]
Yeah, that's a sign that we're already assuming that the `aesara.tensor.linalg` module should cover both, so we'll stick with that.
